tests/unit/components/AddID/AddID.test.js

```diff
@@ -40,11 +56,19 @@ jest.mock( "react-native-vector-icons/MaterialIcons", ( ) => {
   return MaterialIcons;
 } );
 
-describe( "rendering", ( ) => {
+const mockRoute = { params: {} };
+
+describe( "AddID", ( ) => {
+  // Does currently not pass because IconButton does not pass down a11y props
```

Comment:
Yes, accessibilityLabel is working, but the other props I used were stated as missing in the accessibility audit matcher here, so accessibilityValue for example.
I don't remember if I tried to wrap the IconButton in a View with the a11y props, and if that would satisfy the errors.
But also I wasn't sure if we would stick to the IconButton once we have some more design information about those kinds of buttons.

Reply:
Well, what's your opinion? I was thinking we'd continue to use IconButton because I assumed it was probably more accessible than the default button, plus the `mode` prop seems to provide different display styles that correspond to what Abhas has designed, but I don't have strong feelings about it.

Reply:
I did some more work on the IconButton part, turns out mostly it does work with accessibility props added (also the ones not documented). Only in one instance it did not pass down props: When using an icon in a 
`react-native-paper` text input field as the left element, an IconButton is used under the hood. We use the `TextInput.Icon` api for setting the icon, this translated into an IconButton with the icon being used in the button. But I could not get the props to be used in this instance. The new designs do not include this specific problematic icon, so, I would argue for keeping IconButton for now, and look into a more detailed solution if this one instance comes back up at a later time.

Reply:
Got it, so `IconButton` is still ok, but we should be wary of using `TextInput.Icon`.